bpf/include/linux/if_ether.h

```diff
@@ -29,7 +29,7 @@
  */
 
 #define ETH_ALEN	6		/* Octets in one ethernet addr	 */
-#define ETH_HLEN	14		/* Total octets in header.	 */
+#define __ETH_HLEN	14		/* Total octets in header.	 */
```

Comment:
Hm, I missed this earlier, but we should not mangle headers that we sync from upstream kernel. (bpf/include/linux/*)

Reply:
Ouch, then we would need to rename `ETH_HLEN` in our datapath code :-/

Reply:
Hm :/ yeah I guess it would need a `L2_HLEN`...